BanyanArrays/Manifest.toml

```diff
@@ -308,6 +318,18 @@ git-tree-sha1 = "5d494bc6e85c4c9b626ee0cab05daa4085486ab1"
 uuid = "5ced341a-0733-55b8-9ab6-a4889d929147"
 version = "1.9.3+0"
 
+[[MPI]]
```

Comment:
I thought MPI was already added in a previous commit and is in v0.1.3...is there a particular change here?

Reply:
No change in particular, I thought it was added to, maybe the Manifest.toml hasn't been pushed after resolving?